content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -284,20 +302,23 @@ transaction functions. We recommend you get in and out as quick as possible!
 
 </Admonition>
 
-#### Transaction isolation level
+### Transaction isolation level
 
 <Admonition type="info">
 
-This feature is not available on MongoDB, as MongoDB does not support isolation levels.
+This feature is not available on MongoDB, because MongoDB does not support isolation levels.
 
 </Admonition>
 
-From version 4.2.0, you can set the transaction [isolation level](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
+You can set the transaction [isolation level](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels) for transactions in the following Prisma versions:
+
+- For interactive transactions: from version 4.2.0
+- For sequential operations: from version 4.4.0
 
 <Admonition type="info">
 
-**Before version 4.2.0**<br /><br />
-In version 4.1.0 and earlier, you cannot configure the transaction isolation level at a Prisma level. The isolation level is not explicitly set by Prisma, so the isolation level configured in your database is used. For database-specific information on isolation levels, including database defaults, see the following resources:<br /><br />
+**In earlier versions**<br /><br />
+In versions before 4.2.0 (for interactive transactions), or 4.4.0 (for sequential operations), you cannot configure the transaction isolation level at a Prisma level. The isolation level is not explicitly set by Prisma, so the isolation level configured in your database is used. For database-specific information on isolation levels, including database defaults, see the following resources:<br /><br />
```

Comment:
```suggestion
In versions before 4.2.0 (for interactive transactions), or 4.4.0 (for sequential operations), you could not configure the transaction isolation level at a Prisma level. The isolation level is not explicitly set by Prisma, so the isolation level configured in your database is used. For database-specific information on isolation levels, including database defaults, see the following resources:<br /><br />
```

Reply:
Thanks @millsp . The audience for this doc might use any version, so I think we can't tie the wording to a specific version. Also, there is more maintenance overhead if we do that. So for this reason, I'm going to leave the wording as-is, but thanks for the suggestion - it was worth rethinking this.